src/hooks/useTokens.ts

```diff
@@ -74,6 +74,8 @@ export const TokenApiURLs: Record<Chain, string[]> = {
   OP_BNB: [],
   FORM_TESTNET: [],
   CYBER: [],
+  TAIKO: [],
+  KLAYTN: ["https://tokens.coingecko.com/klaytn/all.json"],
```

Comment:
I get an `AccessDenied` error for this 👀

Reply:
Yep, this is wrong indeed. We don't have a token list then